storage-stack-automation/storage_stack.yaml

```diff
@@ -0,0 +1,291 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: "Automatically deploy File Storage Security - Storage stack to every new S3 bucket created and register it to the defined Scanner Stack."
+Metadata:
+  License: MIT-License
+  Version: 1.2
+Resources:
+  lambdarole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      RoleName: !Ref AWS::StackName
+  lambdarolepolicy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyName: !Ref lambdarole
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - "lambda:CreateFunction"
+              - "sqs:DeleteMessage"
+              - "cloudformation:ListExports"
+              - "cloudformation:ListStackInstances"
+              - "sqs:ReceiveMessage"
+              - "iam:CreateRole"
+              - "s3:CreateBucket"
```

Comment:
Would you sort the actions at least by services for better readability?

Reply:
organized the permissions. 
renamed folder structure